README.md

```diff
@@ -222,7 +222,55 @@ client.send(new Error(), { timestamp: new Date(2024, 5, 13, 10, 0, 0) });
 
 ### Customers
 
-New in 0.4: You can set **raygunClient.user** to a function that returns the user name or email address of the currently logged in user.
+You can attach user information to every Raygun Crash Report.
+
+It will be transmitted with the error sent, and a count of affected customers will appear on the dashboard in the error group view.
+If you provide an email address, and the user has associated a Gravatar with it, their picture will be also displayed.
+
+This package offers three different ways to do that:
+
+1. Provide the `userInfo` parameter in the `send()` method.
+2. Implement the `user(request)` method.
+3. Call to the `setUser(user)` method (not recommended and deprecated).
+
+#### User information object
+
+The following properties can be provided as user information:
+
+- `identifier` is the user identifier. This will be used to uniquely identify the user within Raygun. This is the only required parameter, but is only required if you are using customers tracking.
+- `isAnonymous` is a bool indicating whether the user is anonymous or actually has a user account. Even if this is set to true, you should still give the user a unique identifier of some kind.
+- `email` is the user's email address.
+- `fullName` is the user's full name.
+- `firstName` is the user's first or preferred name.
+- `uuid` is the identifier of the device the app is running on. This could be used to correlate user accounts over multiple machines.
```

Comment:
I'm not sure if this should be tracked with the user. In other providers we're not doing this.  To me, the thing that ties users together is `identifier` and if they come from different devices, should this not be set somewhere/somehow differently?

Reply:
No idea, that's how it worked before. This PR didn't change the UserInfo object.

Reply:
Indeed, in the Flutter client, the `uuid` property is not used. Will ask on Slack for clarification.